valhalla/meili/routing.h

```diff
@@ -31,8 +31,6 @@ constexpr uint16_t kInvalidDestination = std::numeric_limits<uint16_t>::max();
 class Label : public sif::EdgeLabel {
 public:
   Label() {
-    // zero out the data but set the node Id and edge Id to invalid
-    memset(this, 0, sizeof(Label));
```

Comment:
This seems dangerous as Label is a child of EdgeLabel, meaning that there would be a vtable which this memset would overwrite. I'm not sure what the memset does since this being the constructor, the fields should already be zero'd.

cc @dnesbitt61

Reply:
it looks like the default constructor for EdgeLabel doesnt zero them out though..

Reply:
Discussed more with @kevinkreiser and the safest solution is probably to initialize the values in the `EdgeLabel` constructor